WordPress/src/main/java/org/wordpress/android/util/config/UnifiedAboutFeatureConfig.kt

```diff
@@ -0,0 +1,24 @@
+package org.wordpress.android.util.config
+
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.annotation.FeatureInDevelopment
+import javax.inject.Inject
+
+/**
+ * Feature configuration for Unified About Screen
+ */
+// TODO: Uncomment the lines @Feature and UNIFIED_ABOUT_REMOTE_FIELD when remote field is configured and
+//  remove line @FeatureInDevelopment and this to-do lines
+// @Feature(DomainPurchaseFeatureConfig.DOMAIN_PURCHASE_REMOTE_FIELD, false)
```

Comment:
This looks like a minor copy/paste typo and it shouldn't really matter for now, but fixing it is really cheap :)

```suggestion
// @Feature(UnifiedAboutFeatureConfig.UNIFIED_ABOUT_REMOTE_FIELD, false)
```